### ISSUE TITLE
Get-AzWebApp returns 'PSSite' object with empty 'kind' property

### ISSUE THREAD

**Issue by arestarh** [Azure/azure-powershell#8841]
## Description
Get-AzWebApp returns 'PSSite' object with empty 'kind' property while Get-AzResource for resourceType "Microsoft.Web/sites" returns 'PSSite' object with 'kind' property with correct value.

## Steps to reproduce

```powershell
Get-AzWebApp -ResourceGroupName "test" -Name "test-web-app"
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      6.1.3
PSEdition                      Core
GitCommitId                    6.1.3
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

## Module versions

```powershell
ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Script     1.4.0      Az.Accounts                         Core,Desk {Disable-AzDataCollection, Disable-AzContextAuto...
Script     1.0.1      Az.Aks                              Core,Desk {Get-AzAks, New-AzAks, Remove-AzAks, Import-AzAk...
Script     1.0.2      Az.AnalysisServices                 Core,Desk {Resume-AzAnalysisServicesServer, Suspend-AzAnal...
Script     1.0.0      Az.ApiManagement                    Core,Desk {Add-AzApiManagementRegion, Get-AzApiManagementS...
Script     1.0.0      Az.ApplicationInsights              Core,Desk {Get-AzApplicationInsights, New-AzApplicationIns...
Script     1.1.2      Az.Automation                       Core,Desk {Get-AzAutomationHybridWorkerGroup, Remove-AzAut...
Script     1.0.0      Az.Batch                            Core,Desk {Remove-AzBatchAccount, Get-AzBatchAccount, Get-...
Script     1.0.0      Az.Billing                          Core,Desk {Get-AzBillingInvoice, Get-AzBillingPeriod, Get-...
Script     1.1.0      Az.Cdn                              Core,Desk {Get-AzCdnProfile, Get-AzCdnProfileSsoUrl, New-A...
Script     1.0.1      Az.CognitiveServices                Core,Desk {Get-AzCognitiveServicesAccount, Get-AzCognitive...
Script     1.5.0      Az.Compute                          Core,Desk {Remove-AzAvailabilitySet, Get-AzAvailabilitySet...
Script     1.0.0      Az.ContainerInstance                Core,Desk {New-AzContainerGroup, Get-AzContainerGroup, Rem...
Script     1.0.1      Az.ContainerRegistry                Core,Desk {New-AzContainerRegistry, Get-AzContainerRegistr...
Script     1.0.2      Az.DataFactory                      Core,Desk {Set-AzDataFactoryV2, Update-AzDataFactoryV2, Ge...
Script     1.0.0      Az.DataLakeAnalytics                Core,Desk {Get-AzDataLakeAnalyticsDataSource, New-AzDataLa...
Script     1.1.0      Az.DataLakeStore                    Core,Desk {Get-AzDataLakeStoreTrustedIdProvider, Remove-Az...
Script     1.0.0      Az.DevTestLabs                      Core,Desk {Get-AzDtlAllowedVMSizesPolicy, Get-AzDtlAutoShu...
Script     1.0.0      Az.Dns                              Core,Desk {Get-AzDnsRecordSet, New-AzDnsRecordConfig, Remo...
Script     1.1.0      Az.EventGrid                        Core,Desk {New-AzEventGridTopic, Get-AzEventGridTopic, Set...
Script     1.0.1      Az.EventHub                         Core,Desk {New-AzEventHubNamespace, Get-AzEventHubNamespac...
Script     1.0.0      Az.HDInsight                        Core,Desk {Get-AzHDInsightJob, New-AzHDInsightSqoopJobDefi...
Script     1.0.2      Az.IotHub                           Core,Desk {Add-AzIotHubKey, Get-AzIotHubEventHubConsumerGr...
Script     1.0.2      Az.KeyVault                         Core,Desk {Add-AzKeyVaultCertificate, Update-AzKeyVaultCer...
Script     1.2.1      Az.LogicApp                         Core,Desk {Get-AzIntegrationAccountAgreement, Get-AzIntegr...
Script     1.0.0      Az.MachineLearning                  Core,Desk {Move-AzMlCommitmentAssociation, Get-AzMlCommitm...
Script     1.0.0      Az.MarketplaceOrdering              Core,Desk {Get-AzMarketplaceTerms, Set-AzMarketplaceTerms}
Script     1.0.0      Az.Media                            Core,Desk {Sync-AzMediaServiceStorageKeys, Set-AzMediaServ...
Script     1.0.1      Az.Monitor                          Core,Desk {Get-AzMetricDefinition, Get-AzMetric, Remove-Az...
Script     1.3.0      Az.Network                          Core,Desk {Add-AzApplicationGatewayAuthenticationCertifica...
Script     1.0.0      Az.NotificationHubs                 Core,Desk {Get-AzNotificationHub, Get-AzNotificationHubAut...
Script     1.1.0      Az.OperationalInsights              Core,Desk {New-AzOperationalInsightsAzureActivityLogDataSo...
Script     1.0.0      Az.PolicyInsights                   Core,Desk {Get-AzPolicyEvent, Get-AzPolicyState, Get-AzPol...
Script     1.0.0      Az.PowerBIEmbedded                  Core,Desk {Remove-AzPowerBIWorkspaceCollection, Get-AzPowe...
Script     1.1.0      Az.RecoveryServices                 Core,Desk {Get-AzRecoveryServicesBackupProperty, Get-AzRec...
Script     1.0.0      Az.RedisCache                       Core,Desk {Remove-AzRedisCachePatchSchedule, New-AzRedisCa...
Script     1.0.0      Az.Relay                            Core,Desk {New-AzRelayNamespace, Get-AzRelayNamespace, Set...
Script     1.2.0      Az.Resources                        Core,Desk {Get-AzProviderOperation, Remove-AzRoleAssignmen...
Script     1.0.0      Az.ServiceBus                       Core,Desk {New-AzServiceBusNamespace, Get-AzServiceBusName...
Script     1.0.1      Az.ServiceFabric                    Core,Desk {Add-AzServiceFabricApplicationCertificate, Add-...
Script     1.0.2      Az.SignalR                          Core,Desk {New-AzSignalR, Get-AzSignalR, Get-AzSignalRKey,...
Script     1.5.0      Az.Sql                              Core,Desk {Get-AzSqlDatabaseTransparentDataEncryption, Get...
Script     1.0.3      Az.Storage                          Core,Desk {Get-AzStorageAccount, Get-AzStorageAccountKey, ...
Script     1.0.0      Az.StreamAnalytics                  Core,Desk {Get-AzStreamAnalyticsFunction, Get-AzStreamAnal...
Script     1.0.1      Az.TrafficManager                   Core,Desk {Add-AzTrafficManagerCustomHeaderToEndpoint, Rem...
Script     1.1.1      Az.Websites                         Core,Desk {Get-AzAppServicePlan, Set-AzAppServicePlan, New...
```

**Comment by markcowl** [comment]
@arestarh Can you provide a debug stream example for this (run Get-AzWebsite showing the issue, with $DebugPreference set to "Continue")?

**Comment by arestarh** [comment]
@markcowl You can find debug stream below. I've only removed all [ADAL] info, renamed subscription ID, App Name and Resource group:

## Debug output
```
DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp?api-version=2018-02-01

Headers:
x-ms-client-request-id        : 7ae07261-9441-4048-a362-513f80ff4af4
Accept-Language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-cache
Pragma                        : no-cache
ETag                          : "1D4D7F322BA1975"
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-request-id               : ad86e5d4-cd5e-4436-8795-9b007c57ef18
Server                        : Microsoft-IIS/10.0
X-AspNet-Version              : 4.0.30319
X-Powered-By                  : ASP.NET
x-ms-ratelimit-remaining-subscription-reads: 11998
x-ms-correlation-request-id   : fbe43a04-4302-47bd-b112-24937d8a439e
x-ms-routing-request-id       : WESTEUROPE:20190326T102602Z:fbe43a04-4302-47bd-b112-24937d8a439e
X-Content-Type-Options        : nosniff
Date                          : Tue, 26 Mar 2019 10:26:02 GMT

Body:
{
  "id": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp",
  "name": "testApp",
  "type": "Microsoft.Web/sites",
  "kind": "api",
  "location": "West Europe",
  "properties": {
    "name": "testApp",
    "state": "Running",
    "hostNames": [
      "testApp.azurewebsites.net"
    ],
    "webSpace": "testRG-WestEuropewebspace",
    "selfLink": "https://waws-prod-am2-119.api.azurewebsites.windows.net:454/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/webspaces/testRG-WestEuropewebspace/sites/testApp",
    "repositorySiteName": "testApp",
    "owner": null,
    "usageState": "Normal",
    "enabled": true,
    "adminEnabled": true,
    "enabledHostNames": [
      "testApp.azurewebsites.net",
      "testApp.scm.azurewebsites.net"
    ],
    "siteProperties": {
      "metadata": null,
      "properties": [
        {
          "name": "LinuxFxVersion",
          "value": ""
        },
        {
          "name": "WindowsFxVersion",
          "value": null
        }
      ],
      "appSettings": null
    },
    "availabilityState": "Normal",
    "sslCertificates": null,
    "csrs": [],
    "cers": null,
    "siteMode": null,
    "hostNameSslStates": [
      {
        "name": "testApp.azurewebsites.net",
        "sslState": "Disabled",
        "ipBasedSslResult": null,
        "virtualIP": null,
        "thumbprint": null,
        "toUpdate": null,
        "toUpdateIpBasedSsl": null,
        "ipBasedSslState": "NotConfigured",
        "hostType": "Standard"
      },
      {
        "name": "testApp.scm.azurewebsites.net",
        "sslState": "Disabled",
        "ipBasedSslResult": null,
        "virtualIP": null,
        "thumbprint": null,
        "toUpdate": null,
        "toUpdateIpBasedSsl": null,
        "ipBasedSslState": "NotConfigured",
        "hostType": "Repository"
      }
    ],
    "computeMode": null,
    "serverFarm": null,
    "serverFarmId": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/serverfarms/wcr-qa",
    "reserved": false,
    "isXenon": false,
    "hyperV": false,
    "lastModifiedTimeUtc": "2019-03-11T10:13:56.0233333",
    "storageRecoveryDefaultState": "Running",
    "contentAvailabilityState": "Normal",
    "runtimeAvailabilityState": "Normal",
    "siteConfig": null,
    "deploymentId": "testApp",
    "trafficManagerHostNames": null,
    "sku": "Standard",
    "scmSiteAlsoStopped": false,
    "targetSwapSlot": null,
    "hostingEnvironment": null,
    "hostingEnvironmentProfile": null,
    "clientAffinityEnabled": true,
    "clientCertEnabled": false,
    "clientCertExclusionPaths": null,
    "hostNamesDisabled": false,
    "domainVerificationIdentifiers": null,
    "kind": "api",
    "outboundIpAddresses": "13.94.143.57,13.94.136.57,40.68.199.146,13.94.138.55,13.94.140.1",
    "possibleOutboundIpAddresses": "13.94.143.57,13.94.136.57,40.68.199.146,13.94.138.55,13.94.140.1,51.144.4.28,52.178.98.215",
    "containerSize": 0,
    "dailyMemoryTimeQuota": 0,
    "suspendedTill": null,
    "siteDisabledReason": 0,
    "functionExecutionUnitsCache": null,
    "maxNumberOfWorkers": null,
    "homeStamp": "waws-prod-am2-119",
    "cloningInfo": null,
    "hostingEnvironmentId": null,
    "tags": null,
    "resourceGroup": "testRG",
    "defaultHostName": "testApp.azurewebsites.net",
    "slotSwapStatus": null,
    "httpsOnly": false,
    "redundancyMode": "None",
    "inProgressOperationId": null,
    "geoDistributions": null
  }
}


DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/web?api-version=2018-02-01

Headers:
x-ms-client-request-id        : 331881ea-98b6-4658-a678-a9092b51fbfc
Accept-Language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-cache
Pragma                        : no-cache
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-request-id               : 8cc9df07-618c-463c-8978-79893c47afe9
Server                        : Microsoft-IIS/10.0
X-AspNet-Version              : 4.0.30319
X-Powered-By                  : ASP.NET
x-ms-ratelimit-remaining-subscription-reads: 11997
x-ms-correlation-request-id   : 42d1e6f9-4996-482a-bb04-88feee30435d
x-ms-routing-request-id       : WESTEUROPE:20190326T102603Z:42d1e6f9-4996-482a-bb04-88feee30435d
X-Content-Type-Options        : nosniff
Date                          : Tue, 26 Mar 2019 10:26:02 GMT

Body:
{
  "id": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/web",
  "name": "testApp",
  "type": "Microsoft.Web/sites/config",
  "location": "West Europe",
  "properties": {
    "numberOfWorkers": 1,
    "defaultDocuments": [
      "Default.htm",
      "Default.html",
      "Default.asp",
      "index.htm",
      "index.html",
      "iisstart.htm",
      "default.aspx",
      "index.php",
      "hostingstart.html"
    ],
    "netFrameworkVersion": "v4.0",
    "phpVersion": "5.6",
    "pythonVersion": "",
    "nodeVersion": "",
    "linuxFxVersion": "",
    "windowsFxVersion": null,
    "requestTracingEnabled": false,
    "remoteDebuggingEnabled": false,
    "remoteDebuggingVersion": null,
    "httpLoggingEnabled": false,
    "logsDirectorySizeLimit": 35,
    "detailedErrorLoggingEnabled": false,
    "publishingUsername": "$testApp",
    "publishingPassword": null,
    "appSettings": null,
    "azureStorageAccounts": {},
    "metadata": null,
    "connectionStrings": null,
    "machineKey": null,
    "handlerMappings": null,
    "documentRoot": null,
    "scmType": "None",
    "use32BitWorkerProcess": true,
    "webSocketsEnabled": false,
    "alwaysOn": false,
    "javaVersion": null,
    "javaContainer": null,
    "javaContainerVersion": null,
    "appCommandLine": "",
    "managedPipelineMode": "Integrated",
    "virtualApplications": [
      {
        "virtualPath": "/",
        "physicalPath": "site\\wwwroot",
        "preloadEnabled": false,
        "virtualDirectories": null
      }
    ],
    "winAuthAdminState": 0,
    "winAuthTenantState": 0,
    "customAppPoolIdentityAdminState": false,
    "customAppPoolIdentityTenantState": false,
    "runtimeADUser": null,
    "runtimeADUserPassword": null,
    "loadBalancing": "LeastRequests",
    "routingRules": [],
    "experiments": {
      "rampUpRules": []
    },
    "limits": null,
    "autoHealEnabled": false,
    "autoHealRules": null,
    "tracingOptions": null,
    "vnetName": "",
    "siteAuthEnabled": false,
    "siteAuthSettings": {
      "enabled": null,
      "unauthenticatedClientAction": null,
      "tokenStoreEnabled": null,
      "allowedExternalRedirectUrls": null,
      "defaultProvider": null,
      "clientId": null,
      "clientSecret": null,
      "clientSecretCertificateThumbprint": null,
      "issuer": null,
      "allowedAudiences": null,
      "additionalLoginParams": null,
      "isAadAutoProvisioned": false,
      "googleClientId": null,
      "googleClientSecret": null,
      "googleOAuthScopes": null,
      "facebookAppId": null,
      "facebookAppSecret": null,
      "facebookOAuthScopes": null,
      "twitterConsumerKey": null,
      "twitterConsumerSecret": null,
      "microsoftAccountClientId": null,
      "microsoftAccountClientSecret": null,
      "microsoftAccountOAuthScopes": null
    },
    "cors": null,
    "push": null,
    "apiDefinition": null,
    "autoSwapSlotName": null,
    "localMySqlEnabled": false,
    "managedServiceIdentityId": null,
    "xManagedServiceIdentityId": null,
    "ipSecurityRestrictions": null,
    "scmIpSecurityRestrictions": null,
    "scmIpSecurityRestrictionsUseMain": false,
    "http20Enabled": false,
    "minTlsVersion": "1.2",
    "ftpsState": "AllAllowed",
    "reservedInstanceCount": 0
  }
}


DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
POST

Absolute Uri:
https://management.azure.com/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/appsettings/list?api-version=2018-02-01

Headers:
x-ms-client-request-id        : 5deed6d4-0eff-47d0-b5f4-e6539ead5061
Accept-Language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-cache
Pragma                        : no-cache
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-request-id               : e8c23a66-5c67-4f6f-b0ea-3ff21235751c
Server                        : Microsoft-IIS/10.0
X-AspNet-Version              : 4.0.30319
X-Powered-By                  : ASP.NET
x-ms-ratelimit-remaining-subscription-resource-requests: 11999
x-ms-correlation-request-id   : 1d632344-7975-4301-9e1d-36c025f83ec8
x-ms-routing-request-id       : WESTEUROPE:20190326T102604Z:1d632344-7975-4301-9e1d-36c025f83ec8
X-Content-Type-Options        : nosniff
Date                          : Tue, 26 Mar 2019 10:26:03 GMT

Body:
{
  "id": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/appsettings",
  "name": "appsettings",
  "type": "Microsoft.Web/sites/config",
  "location": "West Europe",
  "properties": {
    "APPINSIGHTS_INSTRUMENTATIONKEY": "15d8e182-e1ee-4723-85eb-816dfaed3431",
    "ApplicationInsightsAgent_EXTENSION_VERSION": "~2",
    "XDT_MicrosoftApplicationInsights_Mode": "default",
    "DiagnosticServices_EXTENSION_VERSION": "disabled",
    "APPINSIGHTS_PROFILERFEATURE_VERSION": "disabled",
    "APPINSIGHTS_SNAPSHOTFEATURE_VERSION": "disabled",
    "InstrumentationEngine_EXTENSION_VERSION": "disabled",
    "SnapshotDebugger_EXTENSION_VERSION": "disabled",
    "XDT_MicrosoftApplicationInsights_BaseExtensions": "disabled"
  }
}


DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
POST

Absolute Uri:
https://management.azure.com/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/connectionstrings/list?api-version=2018-02-01

Headers:
x-ms-client-request-id        : ab2b7ce4-d038-492e-8cd6-1af1a4cc8150
Accept-Language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-cache
Pragma                        : no-cache
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-request-id               : 29fc4745-7819-4731-89a7-b8bf7c90c50e
Server                        : Microsoft-IIS/10.0
X-AspNet-Version              : 4.0.30319
X-Powered-By                  : ASP.NET
x-ms-ratelimit-remaining-subscription-resource-requests: 11998
x-ms-correlation-request-id   : ce4585fe-b179-460b-9c8d-88e3ef7a2f17
x-ms-routing-request-id       : WESTEUROPE:20190326T102604Z:ce4585fe-b179-460b-9c8d-88e3ef7a2f17
X-Content-Type-Options        : nosniff
Date                          : Tue, 26 Mar 2019 10:26:03 GMT

Body:
{
  "id": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/connectionstrings",
  "name": "connectionstrings",
  "type": "Microsoft.Web/sites/config",
  "location": "West Europe",
  "properties": {}
}


DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
POST

Absolute Uri:
https://management.azure.com/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/azurestorageaccounts/list?api-version=2018-02-01

Headers:
x-ms-client-request-id        : ddbb1139-bd0f-4eff-86c6-7f3a8db6e6c0
Accept-Language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-cache
Pragma                        : no-cache
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-request-id               : 1c6e9b53-cd00-420a-b0b3-952a8a05adb3
Server                        : Microsoft-IIS/10.0
X-AspNet-Version              : 4.0.30319
X-Powered-By                  : ASP.NET
x-ms-ratelimit-remaining-subscription-resource-requests: 11997
x-ms-correlation-request-id   : 20f6230b-3d36-4e09-abc5-c708a1cb6e70
x-ms-routing-request-id       : WESTEUROPE:20190326T102605Z:20f6230b-3d36-4e09-abc5-c708a1cb6e70
X-Content-Type-Options        : nosniff
Date                          : Tue, 26 Mar 2019 10:26:04 GMT

Body:
{
  "id": "/subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/providers/Microsoft.Web/sites/testApp/config/appsettings",
  "name": "azurestorageaccounts",
  "type": "Microsoft.Web/sites/config",
  "location": "West Europe",
  "properties": {}
}




GitRemoteName               :
GitRemoteUri                :
GitRemoteUsername           :
GitRemotePassword           :
AzureStoragePath            : {}
State                       : Running
HostNames                   : {testApp.azurewebsites.net}
RepositorySiteName          : testApp
UsageState                  : Normal
Enabled                     : True
EnabledHostNames            : {testApp.azurewebsites.net, testApp.scm.azurewe
                              bsites.net}
AvailabilityState           : Normal
HostNameSslStates           : {testApp.azurewebsites.net, testApp.scm.azurewe
                              bsites.net}
ServerFarmId                : /subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/prov
                              iders/Microsoft.Web/serverfarms/wcr-qa
Reserved                    : False
IsXenon                     : False
LastModifiedTimeUtc         : 11.03.2019 10:13:56
SiteConfig                  : Microsoft.Azure.Management.WebSites.Models.SiteConfig
TrafficManagerHostNames     :
ScmSiteAlsoStopped          : False
TargetSwapSlot              :
HostingEnvironmentProfile   :
ClientAffinityEnabled       : True
ClientCertEnabled           : False
HostNamesDisabled           : False
OutboundIpAddresses         : 13.94.143.57,13.94.136.57,40.68.199.146,13.94.138.55,13.94.140.1
PossibleOutboundIpAddresses : 13.94.143.57,13.94.136.57,40.68.199.146,13.94.138.55,13.94.140.1,51.144.4.28,52.178.98.21
                              5
ContainerSize               : 0
DailyMemoryTimeQuota        : 0
SuspendedTill               :
MaxNumberOfWorkers          :
CloningInfo                 :
ResourceGroup               : testRG
IsDefaultContainer          :
DefaultHostName             : testApp.azurewebsites.net
SlotSwapStatus              :
HttpsOnly                   : False
Identity                    :
Id                          : /subscriptions/54d2b1a4-bf0a-46d5-9d48-1a088a122c45/resourceGroups/testRG/prov
                              iders/Microsoft.Web/sites/testApp
Name                        : testApp
Kind                        :
Location                    : West Europe
Type                        : Microsoft.Web/sites
Tags                        :

DEBUG: AzureQoSEvent: CommandName - Get-AzWebApp; IsSuccess - True; Duration - 00:00:03.8039353; Exception - ;
DEBUG: Finish sending metric.
DEBUG: 12:26:06 - GetAzureWebAppCmdlet end processing.
DEBUG: 12:26:06 - GetAzureWebAppCmdlet end processing.
```

**Comment by panchagnula** [comment]
@Nking92 can you take a look at this.

**Comment by Nking92** [comment]
Bug has been fixed in Az 2.0.0 release.
